apps/backend/src/mcp/tools/system_monitoring.py

```
@@ -983,6 +962,286 @@ async def get_system_logs(
+        response = {
+            "drives": drives_stats,
+            "filesystem_usage": filesystem_usage,
+            "summary": summary,
+            "device_info": {
+                "hostname": device,
+                "connection_successful": True
+            },
+            "query_info": {
+                "specific_drive": drive,
+                "drives_checked": len(drives_to_check),
+                "timestamp": datetime.now(timezone.utc).isoformat()
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)